src/js/components/stock-movement-wizard/outbound/PickPage.jsx

```diff
@@ -219,6 +219,7 @@ class PickPage extends Component {
       showAlert: false,
       alertMessage: '',
       itemFilter: '',
+      showOnlyErroredItems: false,
```

Comment:
wouldn't errored -> invalid work better?

Reply:
It was also named like that previously, so I would stick to this name at this moment.